src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -4,4 +4,6 @@
 
 public interface RegistrationService {
     User register(User user);
+
+    void clearStorage();
```

Comment:
delete this method

Reply:
![image](https://github.com/mate-academy/jv-registration-validation/assets/114025933/491ac8ae-0375-4ae1-adc4-2648a309262c)

can`t delete this, i am using it in tests